src/components/motor/MotorController.cpp

```diff
@@ -13,7 +13,6 @@ MotorController::MotorController(Controllers::Settings& settingsController) : se
 void MotorController::Init() {
   nrf_gpio_cfg_output(pinMotor);
   nrf_gpio_pin_set(pinMotor);
-  app_timer_init();
```

Comment:
2 notes about app from the NRF SDK:
 - I tend to avoid them because they provide much more functionalities and options than what we need. Also, they will make the port to another MCU (I'm working on the BL602 on the side :) ) more difficult. It's not a big deal on the short term, and this won't prevent me from merging this branch (as we already use it for the motor controller), but on the long term, I think that would be better if we implemented timer from "scratch", probably using timers from FreeRTOS.
 -  As app_timer is already used by the motor driver, did you check these modifications have no side-effects on the vibrations and notifications?

Reply:
i just moved the initialization out of the motor driver and into SystemTask

Reply:
the timer is kind of sketchy and i really don't like how I have to keep track of the time additionally for display purposes, but it's what works for now.

Reply:
moving app_timer_init() out of the motor driver isn't strictly needed, but I found it to be more logical to initialize the timer outside of it now that its no longer exclusively using that functionality.

Reply:
Move app_timer_init() out of the motor driver makes sense, indeed. What I meant is that we should re-test the notification vibrations to ensure these changes did not break it : check that notifications and vibrations still works, still works after the timer was used, still works while the timer is running, ...

Reply:
I don't think there's  any risk of thing breaking here since in principle there isn't any change to the motor controller, but I've tested some things anyways.

- [x] Notifications before the timer is launched once
- [x] Notifications while the timer is running
- [x] Notifications after the timer has stopped